pkg/operator/ceph/pool/controller.go

```diff
@@ -189,6 +190,21 @@ func (r *ReconcileCephBlockPool) reconcile(request reconcile.Request) (reconcile
 
 	updateStatus(r.client, request.NamespacedName, k8sutil.ReconcilingStatus)
 
+	// Get CephCluster version
+	cephVersion, err := opcontroller.GetImageVersion(cephCluster)
+	if err != nil {
+		return reconcile.Result{}, errors.Wrapf(err, "failed to fetch ceph version from cephcluster %q", cephCluster.Name)
```

Comment:
The CI is failing consistently for the helm chart. It's not obvious to me what the issue is, but it seems the pool isn't being created. Could this retrieval of the cephcluster be failing?